practice-two/src/stores/actions.js

```diff
@@ -0,0 +1,31 @@
+import { ACTION_TYPES } from '@constants';
+
+const getListCustomer = (payload) => {
```

Comment:
do you see for all the actions, just difference type, so the type can use the param. It will common function

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/a64db00d-9623-4d26-ab56-3fa6c2b09955)
![image](https://github.com/datdragon777/React-Training/assets/74806826/535d86cf-18dd-468f-b172-3956d7780a7d)